torchx/mix.exs

```diff
@@ -43,4 +44,145 @@ defmodule Torchx.MixProject do
       source_url: @source_url
     ]
   end
+
+  defp aliases do
+    [
+      "compile.torchx": &compile/1
+    ]
+  end
+
+  defp compile(args) do
+    cache_dir = Path.join(__DIR__, "cache")
+
+    force_download = "--force" in args
```

Comment:
@josevalim should we ever react to `--force`? I don't know if this will actually be passed if a user runs `mix compile --force` or just `mix deps.compile --force`  having torchx as a dependency

Reply:
Let's keep this but I don't think it can really be called from a dep. It may be useful for local dev though.

Reply:
Nice. I was worried deps.compile --force or something would be passed into the torchx.compile task. But I just checked and deps.compile --force just removes the dep directory beforehand, so this `--force` won't do any harm outside of torchx development